### DISCUSSION TITLE
Save generated figures as Figure_X.pdf instead of fig-label-X.pdf

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#3360] [Q&A] [answered]
Some academic journals still requires that figures are provided as separate files. For example, [Wiley's Guidelines for the Preparation of Figures](https://authorservices.wiley.com/asset/photos/electronic_artwork_guidelines.pdf) says

> File Naming Convention: to facilitate ease of review, name figure files only with the word “figure” and the appropriate number.

Will be great if Quarto had the option to follow the file naming convention that journals impose.

# Minimal Working Example

``````
---
title: "MWE"
keep-md: true
format:
  docx:
    fig-format: pdf
---

## Results

```{r}
#| echo: false
#| label: fig-mew
#| fig-cap: Awesome figure.

plot(c(1, 2, 3), c(1, 2, 3))
```
``````

The above file is render using `quarto render figure-mwe.qmd`:

```
processing file: figure-mwe.qmd
  |.......................                                               |  33%
  ordinary text without R code

  |...............................................                       |  67%
label: fig-mew (with options) 
List of 2
 $ echo   : logi FALSE
 $ fig.cap: chr "Awesome figure."

  |......................................................................| 100%
  ordinary text without R code


output file: figure-mwe.knit.md

pandoc 
  to: docx
  output-file: figure-mwe.docx
  default-image-extension: png
  
metadata
  title: MWE
  
Output created: figure-mwe.docx
```

## Observed Output

```
$ find figure-mwe_files/
figure-mwe_files/
figure-mwe_files/figure-docx
figure-mwe_files/figure-docx/fig-mew-1.pdf
```

## Desired Output

```
$ find figure-mwe_files/
figure-mwe_files/
figure-mwe_files/figure-docx
figure-mwe_files/figure-docx/figure-1.pdf
```

**Comment by mcanouil** [comment, marked as answer]
Since you are using knitr, you can define a function to pass to `fig.process` option.
The function takes a path and return a path.
In your case input would be "figure-mwe_files/figure-docx/fig-mew-1.pdf" and output would be "figure-mwe_files/figure-docx/figure-1.pdf".
